stdpopsim/genomes.py

```diff
@@ -65,4 +67,12 @@ def recombination_map(self, map_name=None):
         if map_name is None:
             map_name = self.default_genetic_map
         genetic_map = genetic_maps.get_genetic_map(self.species, map_name)
-        return genetic_map.get_chromosome_map(self.name)
+        try:
```

Comment:
I think it would be clearer if this explicitly tested to see if the map exists:
```
if genetic_map.contains_chromosome_map(self.name):
    # same as before
else:
    warnings.warn()
```

Otherwise, I wonder *why* is this raising a ValueError, and why am I defaulting to a 0 map if it's raised?